quandl/get_table.py

```diff
@@ -6,7 +6,7 @@
 import os
 
 
-def get_table(code, **options):
+def get_table(datatable_code, **options):
```

Comment:
we are passing filter conditions in options and column name can be "code". example `quandl.get_table("AT/DATA", code="my_code")`